modules/news/news.graphql.js

```diff
@@ -3,7 +3,7 @@ type News {
 _id: ID!
 title: [Language]
 text: [Language]
-images: [PrimaryImage]
+images: PrimaryImage
```

Comment:
Why have you made this change?

Reply:
It`s ok. I asked him to do so because it will be only one image and we don`t need an array here